src/js/App/Sidenav/Navigation.js

```diff
@@ -84,7 +84,13 @@ const extraLinks = {
 };
 
 const NavItemLink = ({ id, title, external, url, link }) => (
-  <NavItem className="ins-c-navigation__additional-links" key={id} to={url || link} ouiaId={id}>
+  <NavItem
+    className="ins-c-navigation__additional-links"
+    key={id}
+    onClick={() => external && window.open(url || link)}
```

Comment:
You should also include `'_blank','noopener'`
```suggestion
    onClick={() => external && window.open(url || link, '_blank','noopener')}
```